### ISSUE TITLE
Transfer issues and PRs from structureddynamics/Bibliographic-Ontology-BIBO to this repository

### ISSUE THREAD

**Issue by csarven** [dcmi/bibo#1]
https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/ has open issues and PRs. It would be great to pick them up here so that corrections and other improvements can still make their way.

**Comment by nishad** [comment]
@csarven, thanks for raising this issue. Since BIBO at DCMI is not maintained by any working/interest groups, new contributions should go through the DCMI Usage Board instead of direct PRs to this repo. We keep the issues and PR options enabled for this repository for the technical maintenance of BIBO-related files. At this point, DCMI ensures that BIBO has a persistent location and the namespace is well maintained. Please raise your suggestions and updates to the DCMI usage board issue tracker: https://github.com/dcmi/usage/issues

**Comment by niklasl** [comment]
Yes, thank you for this report! We are looking into some more active maintenance of BIBO, and may be able to address some of these going forward.

I'll expand on external issues/requests in this comment, so that we can categorize and prioritize further.

The list of external issues are:

- [bibliographic-ontology.org spends a lot of time down](https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/issues/23)
  - The domain appears abandoned, but (AFAICS) it's not linked to from the official BIBO docs, so probably nothing we can do.
- [No examples](https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/issues/20)
  - To be addressed by https://github.com/dcmi/bibo/pull/2 (and possibly some further documentation work).
- [relax bibo:DocumentPart/dcterms:isPartOf?](https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/issues/24)
- [dcterms:title is declared as an object property, not a data property as in dcterms](https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/issues/21)

The external PR:s are:
- https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/pull/25
- https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/pull/17
- https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/pull/16
- https://github.com/structureddynamics/Bibliographic-Ontology-BIBO/pull/14

There is also this [older list of open issues](https://web.archive.org/web/20100217082232/http://bibliontology.com/nextchanges) at archive.org (from 2010):
- [re-making bibo:identifer a subPropertyOf dcterms:identifier](https://web.archive.org/web/20100217082232/http://bibliontology.com/content/re-making-biboidentifer-subpropertyof-dctermsidentifier)
- [Fix a series of issues introduced by Protégé and some weird imports...](https://web.archive.org/web/20100217082232/http://bibliontology.com/content/fix-series-issues-introduced-prot%C3%A9g%C3%A9-and-some-weird-imports) consisting of:
   1. > The SKOS namespace was the non-reverted ("2008") one.
      - Appears to be fixed since long.
   3. > Every use of rdfs:isDefinedBy (and the two rdfs:seeAlso) have xsd:anyURI-datatyped literals as objects instead of actual URI:s.
      - Tracked here: https://github.com/dcmi/bibo/issues/4
   3. > The (bibo) "abstract" property is also stated to be rdfs:isDefinedBy "http://purl.org/dc/terms/".
      - Also fixed (but see previous point).
   5. > There are a lot of statements about terms from other vocabularies.
      - Tracked here: https://github.com/dcmi/bibo/issues/3
- [Splitting the BIBO files in two: core & law extension](https://web.archive.org/web/20100217082232/http://bibliontology.com/content/splitting-bibo-files-two-core-law-extension) 
- [Adding a bibo_degrees:master individual of the class bibo:ThesisDegree](https://web.archive.org/web/20100217082232/http://bibliontology.com/content/adding-bibodegreesmaster-individual-class-bibothesisdegree)
  - Per https://github.com/dcmi/bibo/issues/3, we may extract the `bibo/status/` and `bibo/degrees/` term lists from the bibo namespace document; but further maintenance/extensions of these is another question.

**Comment by niklasl** [comment]
Closing this, now that we have https://github.com/dcmi/bibo/issues/8 and https://github.com/dcmi/bibo/issues/9 capturing which of the above items we have a plan for doing.

Other than those, the DCMI Usage Board have no plans to address the remaining items listed. They can of course be reported as issues on this repository for future consideration by the DCMI Usage Board. Note that, due to resource constraints, the current process for evolving BIBO requires proposals from active DCMI working groups. (We will strive to resolve errors and mistakes.)
